README.md

```diff
@@ -321,6 +321,15 @@ require 'airbrake/sidekiq'
 If you required Sidekiq before Airbrake, then you don't even have to `require`
 anything manually and it should just work out-of-box.
 
+Airbrake will be notified of all errors from your Sidekiq workers, even when jobs
+will be retried in the future and might work as expected. To filter out these
+errors and only get notified when Sidekiq has exhausted its retries you can
+add the `RetryableJobsFilter` in your app:
```

Comment:
What do you think of the following phrasing:

> By default, Airbrake notifies of all errors, including reoccurring errors during a retry attempt. To filter out these errors and only get notified when Sidekiq has exhausted its retries you can add the `RetryableJobsFilter`:

I think it’s a little shorter and conveys the same meaning.